functions/index.js

```diff
@@ -0,0 +1,163 @@
+//Напиши всі можливі варіанти створення функцій.
+function myFunction() {
+  return;
+} //function declaration
+const test = function () {
+  return;
+}; //function expression
+const add = (a, b) => a + b; //arrow function
+
+//Створи функцію, яка буде виводити кількість переданих їй аргументів.
+function myLength(a) {
+  console.log(myLength.arguments.length);
```

Comment:
arguments are already available in  function 
so u don't. need to write myLength.arguments.length but arguments.length
because you possess that myLength is object ( by writing myLength.arguments.length) but myLength is FUNCTION so you can only call it (like this myLength())